servlet/src/main/scala/org/http4s/servlet/BlockingHttp4sServlet.scala

```diff
@@ -88,4 +100,12 @@ object BlockingHttp4sServlet {
       DefaultServiceErrorHandler,
       dispatcher,
     )
+
+  protected[servlet] def apply[F[_]](
```

Comment:
This one? I think your only recourse to preserve static forwarders is to leave it public and rely on deprecation and source-incompatible changes to dissuade use, as you've done here. Not sure what `protected` does for `object` methods.

Reply:
Note that in Scala 3 it's the other way around. Even package-private object methods will generate static-forwarders.
- https://github.com/lampepfl/dotty/issues/10842

This causes problems with MiMa for internal methods, however.
- https://github.com/lightbend/mima/issues/694

Reply:
Fuck.  I'm just going to leave that one alone then.  They can use the constructor, and Servlet > 3 is coming.

Reply:
What you have here is fine for Scala 2 and 3. The Scala 3 thing works in our favor here, but bites in different situations until MiMa can fix it.